.github/workflows/ci.yml

```diff
@@ -130,3 +144,113 @@ jobs:
           $env:VERUS_Z3_PATH = "$(Get-Location)/z3"; vargo test -p rust_verify_test --test basic
           $env:VERUS_Z3_PATH = "$(Get-Location)/z3"; vargo test -p rust_verify_test --test layout
         shell: powershell
+      - name: build verus release
+        working-directory: .\source
+        run: |
+          ../tools/activate.ps1
+          $env:VERUS_Z3_PATH = "$(Get-Location)/z3"; vargo build --release
+          # $env:VERUS_Z3_PATH = "$(Get-Location)/z3"; ./target-verus/release/verus.exe --version --output-json > ./target-verus/release/version.json
```

Comment:
Why is this commented out?

Reply:
Because I couldn't get it to work, and I don't know why. It's not critical. I'll add a TODO.